pkg/core/function.go

```diff
@@ -53,6 +58,19 @@ func (c *client) CreateFunction(options CreateFunctionOptions, log io.Writer) (*
 		return nil, err
 	}
 
+	labels := s.Spec.RunLatest.Configuration.RevisionTemplate.Labels
+	if labels == nil {
+		labels = map[string]string{}
+	}
+	labels[functionLabel] = options.Name
+	s.Spec.RunLatest.Configuration.RevisionTemplate.SetLabels(labels)
+	annotations := s.Spec.RunLatest.Configuration.RevisionTemplate.Annotations
+	if annotations == nil {
+		annotations = map[string]string{}
+	}
+	annotations[buildAnnotation] = "1"
```

Comment:
it looks like you're attempting to reimplement the generation id. All we really care is that the value is different. I'd rather make it clear that this label has no semantic meaning.

Reply:
the way we use it now, it is a sequence, assuming no one other than the CLI is changing this annotation value